exercises/ruby-1-b/ruby_1_b_test.rb

```diff
@@ -1,34 +1,32 @@
 require 'minitest/autorun'
 require_relative 'ruby_1_b'
 
-# Common test data version: 1.2.0 4fc1acb
-class SentenceTest < Minitest::Test
-  def test_one_word_with_one_vowel
-    assert_equal "a", word_with_most_vowels("a")
+class LibraryTest < Minitest::Test
+  def test_first_book_imported
+    assert Library.books.map(&:isbn).include?("9780191604928")
   end
 
-  def test_one_word_with_one_vowel_and_one_consonant
-    assert_equal "at", word_with_most_vowels("at")
+  def test_last_book_imported
+    assert Library.books.map(&:isbn).include?("9780451524935")
```

Comment:
Assert `assert_equal` provides better messages. Wen this test fails we get an error that says `Expected false to be truthy.`. With something like:
```
assert_equal '9780191604928', Library.books.first.isbn
```
We'll get: 
```
Expected: "9780191604928"
  Actual: "i9780191604928"
```